client/components/mma/accountoverview/SingleContributionCard.tsx

```diff
@@ -0,0 +1,84 @@
+import { css } from '@emotion/react';
+import { space } from '@guardian/source-foundations';
+import { Button, Stack } from '@guardian/source-react-components';
+import { useNavigate } from 'react-router';
+import { convertTimestampToDate } from '../../../../shared/dates';
+import type { SingleProductDetail } from '../../../../shared/productResponse';
+import { convertCurrencyToSymbol } from '../../../utilities/currencyIso';
+import { Card } from '../shared/Card';
+import { productColour } from './ProductCardConfiguration';
+import {
+	buttonLayoutCss,
+	keyValueCss,
+	productDetailLayoutCss,
+	productTitleCss,
+} from './ProductCardStyles';
+
+export const SingleContributionCard = ({
+	singleContributions,
+}: {
+	singleContributions: SingleProductDetail[];
+}) => {
+	const navigate = useNavigate();
+
+	return (
+		<Stack space={4}>
+			<Card>
+				<Card.Header backgroundColor={productColour.singleContribution}>
+					<h3 css={productTitleCss(true)}>Single Support</h3>
+				</Card.Header>
+				<Card.Section>
+					<div
+						css={[
+							productDetailLayoutCss,
+							css`
+								margin-top: -${space[2]}px;
+							`,
+						]}
+					>
+						<dl css={keyValueCss}>
+							{singleContributions.map((contribution) => (
+								<>
+									<div
+										css={css`
+											padding-top: ${space[3]}px;
+										`}
+									>
+										<dt>Single contribution of</dt>
```

Comment:
Really minor but I would check with Helene/UX on where they want to use colons at the end of each statement

Reply:
Yep will do